pallets/payment/src/benchmarking.rs

```diff
@@ -0,0 +1,69 @@
+use super::*;
+
+use crate::Pallet as Payment;
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite, whitelisted_caller};
+use frame_system::RawOrigin;
+use orml_traits::MultiCurrency;
+use virto_primitives::PaymentState;
+
+const SEED: u32 = 0;
+
+fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
+	let events = frame_system::Pallet::<T>::events();
+	let system_event: <T as frame_system::Config>::Event = generic_event.into();
+	// compare to the last event record
+	let frame_system::EventRecord { event, .. } = &events[events.len() - 1];
+	assert_eq!(event, &system_event);
+}
+
+benchmarks! {
+	where_clause { where T::Asset: MultiCurrency<
+		<T as frame_system::Config>::AccountId,
+		CurrencyId = u32, Balance = u32
+	>
+}
+	// create a new payment succesfully
+	create {
+		let caller = whitelisted_caller();
+		let _ = T::Asset::deposit(1u32, &caller, 100u32);
```

Comment:
```suggestion
		let _ = T::Asset::deposit(TEST_ASSET, &caller, 100u32);
```
Maybe I'd prefer this to be a constant or something that describes better what it is since with a `1` is not very clear we are talking about an asset and could look like a quantity. Also, although it probably won't make a big difference, it might give more realistic results to use the actual asset type or something similar to get the benchmarks closer to the real thing, it might be that internally we serialize the asset parameter, store it or do something with it where the `u32` might produce different results that the enum we usually use(e.g. a simple enum in SCALE takes 1 byte vs 4 of the u32).